components/button/Button.js

```diff
@@ -62,15 +60,14 @@ class Button extends PureComponent {
 
     const props = {
       ...others,
-      href,
       ref: node => {
         this.buttonNode = node;
       },
       className: classNames,
       disabled,
       onMouseUp: this.handleMouseUp,
       onMouseLeave: this.handleMouseLeave,
-      type: !href ? type : null,
+      type: element === 'button' ? type : null,
```

Comment:
I think we can actually remove the `type` prop, and just leave it in the `others`.

Reply:
Yeah but, if you include a button in a form element without specifying it's just a regular button, it defaults to a submit button.